addon/components/navbar-auth-dropdown/component.js

```diff
@@ -13,7 +13,8 @@ import config from 'ember-get-config';
  * Sample usage:
  * ```handlebars
  * {{navbar-auth-dropdown
- *   loginAction=loginAction}}
+ *   loginAction=loginAction
+ *   redirectUrl=redirectUrl}}
  * ```
  *
  * @class osf-navbar
```

Comment:
Tiny note: the docstring may need to be updated here to reflect the component name. Otherwise YUIDoc might not build it to the correct docs page.